src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,65 @@
+package com.elixr.training;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            System.out.println("Argument missing");
+            return;
+        }
+        String inputfilepath = args[0];
+        String searchword = args[1];
+        File file = new File(inputfilepath);
+        System.out.println("Processing................");
+        String data = readFile(inputfilepath);
+        isSupportedFile(file);
+        searchword(data, searchword);
+
+    }
+
+    public static String readFile(String fileName) {
+        String data = null;
+        try {
+            data = new String(Files.readAllBytes(Paths.get(fileName)));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return data;
+    }
+
+    public static void isSupportedFile(File file) {
+        if (file.getName().endsWith(".txt") && (file.length() != 0)) {
```

Comment:
Do not hard-code .txt, use a static final constant. And please add support for .json too

Reply:
added the method globally